learn/user-guide/testing-ballerina-code/mocking.md

```diff
@@ -47,7 +47,7 @@ import ballerina/http;
 // An instance of this object can be used as the test double for the `clientEndpoint`.
 public client class MockHttpClient {
 
-    remote function get(@untainted string path, http:RequestMessage message = (), http:TargetType targetType = http:Response) returns @tainted http:Response|http:PayloadType|http:ClientError {
+    remote function get(@untainted string path, map<string|string[]>? headers = (), http:TargetType targetType = http:Response) returns @tainted http:Response|http:PayloadType|http:ClientError {
```

Comment:
We need to update the return too right?

Reply:
Are we validating the return type as well? Because like this it doesn't throw any errors

Reply:
changed to remove `PayloadType`